docs/design-proposals/multi-tenancy/multi-tenancy-network.md

```diff
@@ -238,6 +250,17 @@ Network object specifies service IPAM as *external*:
       service:
         ipam: external
 
+#### User hints
+
+User can provide hints about service IP address.
+
+For Arktos managed pool, it will look at the specific annotation of service object like below
+```arktos.futurewei.com/vip-hint: {"cidr":"10.10.1.0/26"}```
```

Comment:
Got lost.  I was under the impression that this was an optional field in network object, according to our previous discussion.  What's changed and why does this need to be set on every service object?

Reply:
network objects has a list of cidrs for service ip addresses.
Here is for individual service object - if user would like to provide some hints(constraints). It seems implementing such user hints is not of high priority; I'll put them in to TBD, deferring for later decision.